docs/section19/19-24.md

```diff
@@ -0,0 +1,100 @@
+# Section 19 - Rule 19-24         
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-24       
+**Rule Description:** Baseline systems with >= 5,000 CFM supply air and >= 70 %OA shall have energy recovery with 50% enthalpy recovery ratio. The following exceptions apply:
+1. Systems serving spaces that are not cooled and that are heated to less than 60°F. 
+2. Systems exhausting toxic, flammable, or corrosive fumes or paint or dust. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. 
+3. Commercial kitchen hoods (grease) classified as Type 1 by NFPA 96. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. 
+4. Heating systems in Climate Zones 0 through 3. 
+5. Cooling systems in Climate Zones 3C, 4C, 5B, 5C, 6B, 7, and 8. 
+6. Where the largest exhaust source is less than 75% of the design outdoor airflow. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. 
```

Comment:
I think this exception should be applied based on exhaust rates of the zones served by a given baseline system, not based on the zones in the entire building.

Reply:
Function updated with question: For the purposes of finding the largest exhaust source, even if an hvac system has multiple fans I think we would consider that one exhaust source right? The exhaust source would be the sum of the cfm (assuming they are in parallel).